resources/plistbuddy.rb

```diff
@@ -1,34 +1,38 @@
 resource_name :plistbuddy
```

Comment:
Can we call this resource `plist`? `plistbuddy` hints at implementation details that are irrelevant.